tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -207,3 +208,98 @@ def test_infinite_retention(self):
         producer.wait()
         rand_consumer.shutdown()
         rand_consumer.wait()
+
+
+class ShadowIndexingWhileBusyTest(PreallocNodesTest):
+    # With SI enabled, run common operations against a cluster
+    # while the system is under load (busy).
+    # This class ports Test6 from https://github.com/redpanda-data/redpanda/issues/3572
+    # into ducktape.
+    segment_size = 2**30
+    topics = [TopicSpec(partition_count=5)]
+
+    def __init__(self, test_context):
+        self._si_settings = SISettings(
+            log_segment_size=self.segment_size,
+            cloud_storage_cache_size=20 * 2**30,
+            cloud_storage_reconciliation_interval_ms=500,
+            cloud_storage_max_connections=5,
+            cloud_storage_enable_remote_read=False,
+            cloud_storage_enable_remote_write=False)
+
+        super(ShadowIndexingWhileBusyTest,
+              self).__init__(test_context=test_context,
+                             node_prealloc_count=1,
+                             num_brokers=3,
+                             si_settings=self._si_settings)
+
+    @ok_to_fail  # broken pipe inconsistently causes failure from cloud_storage/remote.cc:108
```

Comment:
This should probably be a link to a github issue (create one if it doesn't already exist)